src/modules/reserve-overview/ReserveConfiguration.tsx

```diff
@@ -332,168 +337,38 @@ export const ReserveConfiguration: React.FC<ReserveConfigurationProps> = ({ rese
           <Divider sx={{ my: '40px' }} />
```

Comment:
I like that `<BorrowInfo />` got abstracted out.  Let's do the same for `SupplyInfo`?

Reply:
Done